src/backend/controllers/tankController.js

```diff
@@ -561,20 +561,26 @@ exports.getTankById = async (req: Request, res: Response) => {
 	}
```

Comment:
Currently nothing will be used here for express validation if you don't even have the use of the express validation in the route. As of now, this above code is dead code.